HW5/utils.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
+
+const { PATH } = require("./constants");
+
+const writeDataInFile = (fileName, data) => {
+  fs.writeFileSync(fileName, JSON.stringify(data), "utf8", (err) =>
```

Comment:
FYI: you can structurize your json data in file using JSON.stringify 3rd argument: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#space

Reply:
Updated